### PR TITLE
Router health check notification mail to show router name next to UUID

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...

Fixes: #6129

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [x] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by rohityadavcloud — APPROVED** [review]

File: server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java [hunk 1]
```diff
@@ -1215,7 +1215,7 @@ private void handleFailingChecks(DomainRouterVO router, List<String> failingChec
             return;
         }
 
-        String alertMessage = "Health checks failed: " + failingChecks.size() + " failing checks on router " + router.getUuid();
+        String alertMessage = String.format("Health checks failed: %d failing checks on router %s (UUID:%s)", failingChecks.size(), router.getName(), router.getUuid());
```
Comment (rohityadavcloud) [inline server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java:1218]: minor nit - we should just refer UUID with ID for user-facing strings. LGTM.

File: server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java [hunk 2]
```diff
@@ -1215,7 +1215,7 @@ private void handleFailingChecks(DomainRouterVO router, List<String> failingChec
...
         _alertMgr.sendAlert(AlertType.ALERT_TYPE_DOMAIN_ROUTER, router.getDataCenterId(), router.getPodIdToDeployIn(),
                 alertMessage, alertMessage);
         s_logger.warn(alertMessage + ". Checking failed health checks to see if router needs recreate");
```
Comment (shwstppr) [inline server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java:1221]: @DaanHoogland should we add name in alert message (generated below) as well?

Comment (DaanHoogland) [reply server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java:1221]: yes, we should thanks

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by sureshanaparti — APPROVED** [review]
code LGTM

**Review by Pearl1594 — APPROVED** [review]

**Review by weizhouapache — APPROVED** [review]

**Pull request merged**